clang/lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -1977,6 +1977,22 @@ static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
                                   D2->getTemplatedDecl()->getType());
 }
 
+static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
+                                     TypeAliasTemplateDecl *D1,
+                                     TypeAliasTemplateDecl *D2) {
```

Comment:
This function should work if similar to the one with `ClassTemplateDecl` (use `IsTemplateDeclCommonStructurallyEquivalent`).